modules/library/src/main/scala/zio/elasticsearch/executor/HttpExecutor.scala

```diff
@@ -403,32 +403,31 @@ private[elasticsearch] final class HttpExecutor private (esConfig: ElasticConfig
     searchAfter: Option[Json]
   ): Task[(Chunk[Item], Option[(String, Option[Json])])] = {
     val pointInTimeJson =
-      Json.Obj(
-        "pit" -> Json.Obj(
-          "id"      -> Json.Str(pitId),
-          KeepAlive -> Json.Str(config.keepAlive)
+      Obj(
+        "pit" -> Obj(
+          "id"      -> Str(pitId),
+          KeepAlive -> Str(config.keepAlive)
```

Comment:
Can you replace` Str(...) `with `.toJson`? Everywhere where is possible in this file.